src/main/java/com/google/sps/templates/mapPages.soy

```diff
@@ -112,6 +112,7 @@
                 <div class="mdc-checkbox">
                     <input type="checkbox" class="mdc-checkbox__native-control" checked name="groupCheckbox" 
                     value={$groupId} onclick="toggleGroupMarkers(this, {$groupId})">
+                        <script>getMarkerInfo({$groupId})</script>
```

Comment:
is there a reason this is in a script tag? can it not be in the mapScript.js?

Reply:
I wanted it call the function in mapScript.js one time for each group when the the checkboxes were getting instantiated